lib/beaker/host.rb

```diff
@@ -499,6 +501,7 @@ def do_rsync_to from_path, to_path, opts = {}
     'freebsd',
     'windows',
     'pswindows',
+    'eos',
```

Comment:
@rick, is there a reason in Ruby why this should be at the bottom of the file?  Like how javascript makes the page load faster by being at the bottom kinda thing?

This was a little painful to find for me, so I'm wondering if there's any reason that I can't move this to the top like every other require ever.